source/getting-started/build-host-setup.rst

```diff
@@ -13,17 +21,26 @@ Install the following software:
 * NASM
 * OpenSSL
 
-You can also consider Docker containers to build |SPN|. See :ref:`build-on-docker` for more details.
 
+Build Tools Download - Ubuntu
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Install required packages on Ubuntu::
 
-.. note:: Python3 is not supported.

```

Comment:
Why to remove this? IMO, may be its clearer this way that Python 3 is indeed not supported.

Reply:
Support Python version has been explicitly stated. Will add support for Python 3 shortly.